datus/storage/metric/store.py

```
@@ -174,6 +174,101 @@ def search_all(
+            "yaml_updated": False,
+        }
+
+        # Handle yaml file if yaml_path exists
+        if yaml_path and os.path.exists(yaml_path):
+            try:
+                # Read yaml file (supports multi-document format)
+                with open(yaml_path, "r", encoding="utf-8") as f:
+                    docs = list(yaml.safe_load_all(f))
+
+                # Filter out the metric doc with matching name
+                original_count = len(docs)
```

Ruff (0.14.13)

Local variable `original_count` is assigned to but never used

Remove assignment to unused variable `original_count`

(F841)